sourcefiles/treasures/treasurewriter.py

```diff
@@ -128,6 +133,48 @@ def get_treasure_tier_dict(settings: rset.Settings):
     #       Let's just handle difficulty by moving boxes up/down in tier
     return treasure_tier_dict
 
+# if rset.GameFlags.TREASURE_TIER_MARKERS in settings.gameflags:
+def write_treasure_tier_markers(ct_rom: ctrom.CTRom,
+                                settings: rset.Settings):
+                
+    treasure_tier_dict = get_treasure_tier_dict(settings)
+    treasure_data = tt.get_base_treasure_dict()
+    treasure_marker_dict = td.get_treasures_tier_marker_dict()
+
+    for tier,marker in treasure_marker_dict.items():
+        treasures = treasure_tier_dict[tier]
+
+        for tid in treasures:
+            chest = treasure_data[tid]
+            chest_data = chest.get_chest_data(ct_rom)
+            location_id = tt.get_chest_loc_id(chest.chest_index)
+
+            # get script
+            script = ct_rom.script_manager.get_script(location_id)
+            obj_id = script.append_empty_object()
+            x_coord = (chest_data.x_coord << 4) + 10
```

Comment:
Treasure coordinates are tile coordinates, so you should just use `EC.set_object_coordinates_tile` instead of repeating a conversion calculation like the one in `EC.set_object_coordinates_auto`.  When I did this, tile coordinates worked perfectly.

Reply:
Alright, will give it a try